app/templates/components/nav-bar.hbs

```diff
@@ -1,9 +1,20 @@
+<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" rel="stylesheet"/>
 <div class="mobile hidden row">
   <div class="ui menu">
     <LinkTo
       @route="index" class="header item" @activeClass="">
       <h3>{{this.settings.appName}}</h3>
     </LinkTo>
+    <div class="ui container" style="max-width: 400px;">
+      <div class="ui input" style="padding: 10px; margin-left: 160px; min-width: 900px;">
```

Comment:
pls use p, ml classes for this

Reply:
The rest of the reviewers told me semantic UI will be a better option

Reply:
![Screenshot (225)](https://user-images.githubusercontent.com/47476857/102122183-22ba7580-3e6b-11eb-9ab7-78b5db1f8f14.png)
![Screenshot (226)](https://user-images.githubusercontent.com/47476857/102122190-264dfc80-3e6b-11eb-8aff-f01d0319b626.png)

Reply:
no this is used instaed of using inline styling these are css classes for padding, margin

Reply:
Please check @iamareebjamal @daretobedifferent18

Reply:
> no this is used instead of using inline styling these are CSS classes for padding, margin

Yeah! I understand but this will not go with us icon input and have to make several divs to have the same UI

Reply:
@mariobehling is this correct?

Reply:
Hi, can you add separate .SCSS file in style folder?